data/zakariyahrobiat.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Zakariyah Robiah",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I'm a food scientist turned frontend developer",
+  "avatar": "https://github.com/ZakariyahRobiat.png"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/zakariyahrobia3",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/zakariyahrobiah",
```

Please fix this invalid url. You've mistyped your username here.

```suggestion
      "url": "https://github.com/zakariyahrobiat",
```

---

I can see your GitHub username has a "t" at the end rather than a "h" so I will commit this suggestion